services/director-v2/src/simcore_service_director_v2/modules/rabbitmq.py

```diff
@@ -25,6 +49,13 @@ async def on_startup() -> None:
             client_name="director-v2", settings=settings
         )
 
+        await app.state.rabbitmq_client.subscribe(
+            WalletCreditsLimitReachedMessage.get_channel_name(),
+            partial(message_handler, app),
+            exclusive_queue=False,
+            topics=[f"*.{CreditsLimit.SHUTDOWN_SERVICES}"],
```

Comment:
Q: this will always be `*.0`? 
Perhaps I got it wrong but I have the impression that the Enum trick is harming us. 
Why do we use the threshold value to name message?

Reply:
Yes, you are both right. Generally, it should work because the change will be propagated, but yes I didn't realize that it might be nicer to have some reasonable default "string" value

Reply:
so we leave it as is for now